docs/blog/index.md

```diff
@@ -1,5 +1,15 @@
 # ORAS Blog Index
 
+<h2>ORAS Welcomes New Owners, Maintainers, and Thanks Emeritus Owners</h2>
+
+_ORAS Community_
+
+With a supermajority vote from the existing ORAS organization owners, we are excited to announce that [Terry Howe](https://github.com/TerryHowe) and [Andrew Block](https://github.com/sabre1041) have been nominated and accepted as new owners of the ORAS project in Mar 2023. In addition, ORAS also has two contributors [Billy Zha](https://github.com/qweeah) and [Feynman Zhou](https://github.com/feynmanzhou) been nominated and accepted as subproject maintainers in Feb 2023.
```

Comment:
A few tweaks to tighten it up. To be "accepted," it's assumed they're "nominated", and the specific dates are interesting, but just made it a long sentence. 
The blog should likely have a date that associates a time to the announcement. 

```suggestion
With a supermajority vote from the existing ORAS Organization Maintainers, we are excited to announce that [Terry Howe](https://github.com/TerryHowe) and [Andrew Block](https://github.com/sabre1041) have been accepted as new ORAS Organization Maintainers. In addition, [Billy Zha](https://github.com/qweeah) and [Feynman Zhou](https://github.com/feynmanzhou) been accepted as ORAS subproject maintainers.
```

Reply:
@SteveLasker From the [governance doc](https://github.com/oras-project/community/blob/main/governance/GOVERNANCE.md#oras-org-owners), ORAS only has Organization Owners but no Organization Maintainers defined. To align the role name with the governance doc, shall we use Organization Owners here?